app/models/user.rb

```diff
@@ -15,6 +15,9 @@ class User < ActiveRecord::Base
   # rater
   ratyrate_rater
 
+  # public activity
```

Comment:
Consider removing comment on line 18, it duplicates information the code on line 19 explains (that this is part of public activity).